gradle.properties

```diff
@@ -12,7 +12,7 @@ untilBuildVersion = 221.*
 pluginVerifierIdeVersions = 2021.1, 2021.3, 2022.1
 
 platformType = IC
-platformVersion = LATEST-EAP-SNAPSHOT
+platformVersion = 2022.1.3
```

Comment:
I'll eventually get around to releasing the version of AMII that supports the 2022.2 builds. Until then, will put this back so the CI can pass.